examples/Factory_method/factory_method_example_1.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+from abc import ABCMeta, abstractmethod
+
+
+class GameClass(metaclass=ABCMeta):
```

Comment:
요 방식이 권장됩니다. 
https://docs.python.org/3/whatsnew/3.4.html#abc 
```suggestion
from abc import ABC, abstractmethod


class GameClass(ABC):
```